src/change_stream.ts

```diff
@@ -243,6 +246,8 @@ export class ChangeStream<TSchema extends Document> extends TypedEventEmitter<Ch
   ) {
     super();
 
+    this.isIterator = false;
+    this.isEmitter = false;
```

Comment:
I think it'd be good to consolidate these and use a Symbol for the property, e.g. `this[kMode] = false|'iterator'|'emitter'`;